src/Microsoft.Azure.Batch.SoftwareEntitlement/EntitlementVerificationRequest.cs

```diff
@@ -0,0 +1,36 @@
+using System.Net;
+
+namespace Microsoft.Azure.Batch.SoftwareEntitlement
+{
+    /// <summary>
+    /// The information provided by (or inferred about) the application and machine making the request.
+    /// </summary>
+    /// <remarks>
+    /// This doesn't include the token or any claims associated with it, which is assumed to have been
+    /// provided by Batch and passed on untouched by the application. This is the data that is compared
+    /// against the token's claims to determine whether the request is allowed.
+    /// </remarks>
+    public class EntitlementVerificationRequest
```

Comment:
Is it the intention that `EntitlementVerificationRequest` will be extended with **max-cores** and **host-id** for the next version of the protocol?

Reply:
Yes